languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,54 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_QUE" name="quelle que" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token>qu'</token>
+                      <token>elle</token>
+                  </marker>
+                  <token regexp="yes">que|qu'</token>
+              </pattern>
+              <message>Dans cette phrase, "qu'elles" est une erreur grammaticale. Souhaitez-vous écrire <suggestion>quelle</suggestion> ?</message>
```

Comment:
The message below about a possible pronoun error is much better than the "erreur grammaticale". It could be used here as well!